blocks/trending-news/trending-news.js

```diff
@@ -0,0 +1,113 @@
+import { getTrendingNews } from '../../scripts/mockapi.js';
+
+export default function decorate(block) {
+  const newsData = getTrendingNews();
+  block.textContent = '';
+
+  function createNewsItem(news) {
+    const article = document.createElement('div');
+    article.className = 'article';
+
+    const mediaWrapper = document.createElement('div');
+    mediaWrapper.className = 'picture-wrapper';
+
+    const picture = document.createElement('picture');
+    const img = document.createElement('img');
+    img.src = news.imgUrl;
+    img.alt = '';
+    img.loading = 'lazy';
+    picture.appendChild(img);
+    mediaWrapper.appendChild(picture);
+
+    const textContent = document.createElement('div');
+    textContent.className = 'article-text';
+
+    const h3 = document.createElement('h3');
+    h3.className = 'post_title';
+    const a = document.createElement('a');
+    a.href = news.link;
+    a.target = '_blank';
+    a.tabIndex = '0';
+    a.textContent = news.title;
+    h3.appendChild(a);
+    textContent.appendChild(h3);
+
+    const postMeta = document.createElement('div');
+    postMeta.className = 'post-info';
+    const imgTime = document.createElement('img');
+    imgTime.src = 'https://www.icicidirect.com/Content/images/time.svg';
```

Comment:
This should ideally be placed as icon on the page. See `decorateIcons`

Reply:
Agreed, Done